torch/_inductor/codegen/triton.py

```diff
@@ -1896,7 +1896,7 @@ def dense_size_str(self):
                 sizes.append("1")
         return f"[{', '.join(sizes)}]"
 
-    def call_kernel(self, name: str):
+    def call_kernel(self, name: str, node: IRNode = None):
```

Comment:
Let's make node required and update the users.  We can drop `name` since there is `node.get_name()`.

Reply:
Actually name != node.get_name(). name here is generated by `self.define_kernel(src_code, node_schedule)`.